pkg/operator/starter.go

```diff
@@ -232,6 +255,14 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		return err
 	}
 
+	oauthEncryptionController := oauthapiencryptioncontroller.New(
```

Comment:
this is just the secret sync controller, isn't it? Can we make that clearer?

Reply:
sure